tests/callbacks/test_stochastic_weight_avg.py

```diff
@@ -38,17 +44,17 @@ def __init__(self, batchnorm: bool = True, interval: str = "epoch", iterable_dat
         self.layer = nn.Sequential(*layers)
         self.interval = interval
         self.iterable_dataset = iterable_dataset
+        self.crash_after_epoch = crash_after_epoch
+        self.save_hyperparameters()
```

Comment:
Is adding `save_hyperparameters` relevant?

Reply:
No I don't remember why I put this in but it doesn't appear to be necessary now, I'll remove it.